container_files/public_html/doc/builtin/ClassifierConf.md

```diff
@@ -132,6 +133,15 @@ all the values in two, using the threshold that provides the best separation
 of classes. You can always do you own discretization, for instance using a
 `CASE` expression.
 
+<a name="fasttext"></a>
+### FastText (type=fasttext)
+
+![](%%jmlclassifier fasttext)
+
+Note that our version of the Fast Text Classifier only supports feature counts, and currently
```

Comment:
Je veux dire que FastText prend des bag of words. La valeur numérique de la feature sera toujours interprétée comme le nombre de fois que la feature apparait dans l'exemple.